### ISSUE TITLE
Problem with JulesJordan

### ISSUE THREAD

**Issue by PAHelper2** [DirtyRacer1337/Jellyfin.Plugin.PhoenixAdult#52]
**Describe the bug**
Identify finds the movie properly, but then fails to update the metadata.

**Include**
1. Jellyfin 10.7.1
2. plugin version 2.5.1.43
3. https://www.julesjordan.com/trial/scenes/Angela-White-Gabbie-Carter-Big-Natural-Tits-Busty-Threesome-Teen_vids.html
4. JulesJordan.21.03.30.Angela.And.Gabbie.XXX.MP4-P0RNL0V3RSD.mp4
5. Log:
```
,[17:58:13] [INF] [69] PhoenixAdult.Provider: searchInfo.Name: JulesJordan - Angela.White.Gabbie.Carter
,[17:58:13] [INF] [69] PhoenixAdult.Provider: site: 25:0 (JulesJordan)
,[17:58:13] [INF] [69] PhoenixAdult.Provider: searchTitle: Angela White Gabbie Carter
,[17:58:13] [INF] [69] PhoenixAdult.Provider: searchDate: 
,[17:58:13] [INF] [69] PhoenixAdult.Provider: provider: PhoenixAdult.Sites.SiteJulesJordan
,[17:58:17] [WRN] [69] Jellyfin.Server.Middleware.ResponseTimeMiddleware: Slow HTTP Response from http://foobarserver/Items/RemoteSearch/Movie to 172.17.0.1 in 0:00:03.9322215 with Status Code 200
,[17:58:20] [INF] [114] Jellyfin.Api.Controllers.ItemLookupController: Setting provider id's to item 0ab91ecd-bbb6-6b40-6ee9-03d404055b65-JulesJordan.21.03.30.Angela.And.Gabbie.XXX.MP4-P0RNL0V3RSD: {"PhoenixAdult":"25#0#P9GMNCGWDyH7WG9yJJzAMvW64729y74UXbKSULBHqsArn57wvzRSKmnHKC3mwJyECCtuXiV7VHaZr5UDoGJCPtSshTrqSTtrukTXo2nfr9g1rXVMWvieixew#2021-03-30"}
,Unsupported codec with id 100359 for input stream 2
,[17:58:21] [INF] [114] PhoenixAdult.Provider: PhoenixAdult ID: 25#0#P9GMNCGWDyH7WG9yJJzAMvW64729y74UXbKSULBHqsArn57wvzRSKmnHKC3mwJyECCtuXiV7VHaZr5UDoGJCPtSshTrqSTtrukTXo2nfr9g1rXVMWvieixew#2021-03-30
**,[17:58:25] [ERR] [102] PhoenixAdult.Provider: Update error: "System.NullReferenceException: Object reference not set to an instance of an object.**
,   at PhoenixAdult.Sites.SiteJulesJordan.Update(Int32[] siteNum, String[] sceneID, CancellationToken cancellationToken)
,   at PhoenixAdult.Provider.GetMetadata(MovieInfo info, CancellationToken cancellationToken)"
,[17:58:36] [WRN] [93] Jellyfin.Server.Middleware.ResponseTimeMiddleware: Slow HTTP Response from http://foobarserver/Items/RemoteSearch/Apply/0ab91ecdbbb66b406ee903d404055b65?ReplaceAllImages=true to 172.17.0.1 in 0:00:15.5168322 with Status Code 204
```

**Comment by DirtyRacer1337** [comment]
Fixed in 3df3b3aab8a143ed6eedf124c5f728bc29f755b4

**Comment by PAHelper2** [comment]
Fast work!  Thanks!
